cdap-docs/examples-manual/source/examples/purchase.rst

```diff
@@ -9,47 +9,47 @@
 Purchase
 ========
 
-A Cask Data Application Platform (CDAP) Example demonstrating many of the CDAP components.
+A Cask Data Application Platform (CDAP) example demonstrating many of the CDAP components.
 
 Overview
 ========
 
-This example demonstrates use of many of the CDAP components—Streams, Flows, Flowlets,
-Datasets, Queries, MapReduce programs, Workflows, and Services—all in a single Application.
+This example demonstrates use of many of the CDAP components |---| streams, flows, flowlets,
+datasets, queries, MapReduce programs, workflows, and services |---| all in a single application.
```

Comment:
No, because here we are talking about a SQL query, which we show an example of using the Explore feature of CDAP.